src/core/runtime.hpp

```diff
@@ -236,26 +236,6 @@ activateBlock(CBlock *blk, CBContext *context, const CBVar &input) {
     auto cblock = reinterpret_cast<chainblocks::BlockWrapper<Input> *>(blk);
     return cblock->block.activate(context, input);
   }
-  case CoreTakeSeq: {
```

Comment:
This is the place where inlining optimizations might happen as mentioned before at Enum, they might be worth for Seq, String, Bytes to avoid indirections